### ISSUE TITLE
"Language" Accessibility Option updates

### ISSUE THREAD

**Issue by EvanSchallerer** [osu-cass/SampleItemsWebsite#165]
- "Language" Accessibility option label should be "Translations (Stacked)"
- Move to category "Designated Support"

**Comment by alexlepinski** [comment]
done
